configs/default/GEPR-TAKERG4AIO.config

```diff
@@ -91,7 +91,6 @@ feature AIRMODE
 
 
 # master
-set baro_i2c_device = 1
```

Comment:
Why remove baro i2c device setting?

Reply:
removed baro drivers, as no baro is present on final version.